numpy/random/_pcg64.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Dict, Literal, Union
+
+from numpy.random.bit_generator import BitGenerator as BitGenerator
+from numpy.random.bit_generator import SeedSequence as SeedSequence
+from numpy.typing import _ArrayLikeInt_co
+
+class PCG64(BitGenerator):
+    def __init__(
+        self, seed: Union[None, int, _ArrayLikeInt_co, SeedSequence] = ...
```

Comment:
```suggestion
        self, seed: Union[None, _ArrayLikeInt_co, SeedSequence] = ...
```
Nit: `int` is already included in the definition of `_ArrayLikeInt_co`.

Reply:
Fixed, thanks.